src/main/java/core/basesyntax/servise/ReportWriterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.servise;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.servise.inrterfase.ReportWriter;
+import core.basesyntax.storage.Storage;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ReportWriterImpl implements ReportWriter {
+    @Override
+    public void writeReport(String fileName) {
+        File newFile = new File(fileName);
+        try (BufferedWriter bufferedWriter =
+                     new BufferedWriter(new FileWriter(newFile, true))) {
+            bufferedWriter.write("fruit,quantity" + System.lineSeparator());
```

Comment:
Let's create constant for "fruit,quantity"

Reply:
done